docs/wiki/Getting started - Scenario 1 Consume library.md

```diff
@@ -121,8 +121,14 @@ To publish a module by running the script:
          ```PowerShell
         Publish-ModuleToPrivateBicepRegistry -TemplateFilePath "D:\ResourcesModules\modules\Microsoft.KeyVault\vaults\deploy.bicep" -ModuleVersion "0.4.740" -BicepRegistryName 'adpsxxazacrx001'  -BicepRegistryRgName 'artifact-rg'
         ```
-    As the modules to be published are more than one a script that calls the `'Publish-ModuleToPrivateBicepRegistry'` function for each of the modules can be created.
+    If you need to publish more than one module, you could use the following PowerShell script that calls the `'Publish-ModuleToUniversalArtifactsFeed'` function for each of the modules:
```

Comment:
Hey @oliverlabs, thanks for the contribution. Could you do us the favor and replicate the snippet also for the template specs & artifacts feed?